### DISCUSSION TITLE
Bad request for storage.from.list()

### DISCUSSION THREAD

**Discussion by bukulele** [supabase/supabase#7326] [Questions] [answered]
Today suddenly I've started to receive an error with the message "Body cannot be empty when content-type is set to 'application/json'" as a response to request via storage.from.list() method. I have not changed any code.
The strange is this error occurs ONLY when I logged under one exact user. If I'm not logged at all OR if I logged under another user, there is no error and my website receives the data successfully. 

Any ideas what could happen and how to fix this?

**Comment by GaryAustin1** [comment, marked as answer]
Possibly this: https://status.supabase.com/incidents/mfz0jh87pd7z

**Comment by steve-chavez** [comment]
Please retry the request, this could have been related to https://github.com/supabase/supabase/issues/7293, which was recently fixed.
